### DISCUSSION TITLE
加上--fullscreen参数，但是MSI运行起来并没有自动全屏。。。

### DISCUSSION THREAD

**Discussion by drw2005** [tw93/Pake#599] [Q&A]
打包成功，但是并没有全屏运行，是命令不对吗？谢谢！

pake ./clock/index.html --name clock --fullscreen

**Comment by LinChenRui** [comment]
windows打包我也发现这个问题，改了一下构建窗口的代码，这里我强行使用全屏，你可以根据配置文件全屏状态判断是否开启全屏，如图所示：
![20240607151704](https://github.com/tw93/Pake/assets/69792595/2e2a98b2-5c52-4b56-a31c-af297589292e)
